src/geefetch/data/get.py

```diff
@@ -102,6 +106,9 @@ def download_chip_ts(
             **kwargs,
         )
         log.debug(f"Succesfully downloaded chip to [cyan]{out}[/]")
+    except ValueError as e:
+        log.error(f"ValueError in download_chip_ts for {out}: {e}")
+        raise DownloadError from e
```

Comment:
Is this to improve error logging ? this seems unnecessary compared to catching it with `except Exception`